ethereum_history_api/circuits/lib/src/misc/u128_to_u8_test.nr

```diff
@@ -0,0 +1,12 @@
+use crate::misc::u128_to_u8::{u128_to_u8};
+
+#[test]
+fn test_u128() {
```

Comment:
Test cases:
- zero
- just low
- just high
- max

Reply:
I've thought about those tests and the underlying function u64_to_u8 has good test coverage so here we just need to check that we concat results in the correct order. This single test that I've added does exactly that